client/state/lib/middleware.js

```diff
@@ -91,6 +95,22 @@ const updateSelectedSiteForDesktop = ( dispatch, action, getState ) => {
 	desktop.setSelectedSite( selectedSite );
 };
 
+/*
+ * Here be dragons.
+ */
```

Comment:
I think it's fine leaving it here, we just need a little context with what it's doing. 

If you think it's clearer to move this out to lib/sites-once we'd probably need to add a lint rule to prevent new usages.